LocalStorage/Models/CityRealmObject.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CityRealmObject.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 05/04/2024.
+//
+
+import Foundation
+import RealmSwift
+
+/// london Array<weatherInfo 20-10-2024>
+/// each city will have an array of weather history info
+final class CityRealmObject: Object, ObjectKeyIdentifiable {
+    @Persisted var cityName: String
+    @Persisted var weatherInfoList = List<FormattedCityWeatherModel>()
+
+    convenience init(weatherInfoItem: FormattedCityWeatherModel) {
+        self.init()
+        
+        self.weatherInfoList.append(weatherInfoItem)
+        self.cityName = weatherInfoItem.cityName
+    }
+
+    func getLocalStorageCity() -> LocalStorageCity {
```

Comment:
```suggestion
    func asLocalStorageCity() -> LocalStorageCity {
```

This method doesn't actually retrieve as list, but instead convert the realm model to a business model.